packages/fluentui/react-bindings/test/icons/createSvgIcon-test.tsx

```diff
@@ -41,12 +41,35 @@ describe('createSvgIcon', () => {
   });
 
   it('spreads unhandled props on the root element', () => {
-    const TestIcon = createSvgIcon<SvgIconProps>({ svg: testSvg, displayName: 'TestIcon' });
+    const TestIcon = createSvgIcon({ svg: testSvg, displayName: 'TestIcon' });
 
     const wrapper = mount(<TestIcon id="test-id" />, {
       wrappingComponent: TestProvider,
       wrappingComponentProps: { theme: createTheme() },
     });
-    expect(wrapper.find(`.${SvgIconClassName}`).props().id).toEqual('test-id');
+    expect(wrapper.find(`.${svgIconClassName}`).props().id).toEqual('test-id');
+  });
+
+  it('provides all props on the svg function', () => {
+    interface BookProps extends SvgIconProps {
```

Comment:
```suggestion
    interface BookProps {
```

_I_ expect that `BookProps` should be enough